test/test_Bassa_endpoint.py

```diff
@@ -0,0 +1,35 @@
+import unittest
+import requests
+
+headers = {
+'Host': 'localhost:5000',
+'Accept': 'application/json, text/plain, */*',
+'Accept-Language': 'en-US,en;q=0.5',
+'Accept-Encoding': 'gzip, deflate',
+'Referer': 'http://localhost:3000/',
+'Content-Type': 'application/x-www-form-urlencoded',
+'Content-Length': '28',
+'Origin': 'http://localhost:3000',
+'Connection': 'close'
+}
+payload = {''}
+class TestFlaskAPIUsingRequests(unittest.TestCase):
+    def test_api_for_404(self):
```

Comment:
Let's remove this test as it doesn't test for any functionality

Reply:
done